test/views/html/propertyDetails/PropertyDetailsTaxAvoidanceSpec.scala

```diff
@@ -68,9 +68,14 @@ class PropertyDetailsTaxAvoidanceSpec extends AtedViewSpec {
       errorDoc.getElementsMatchingOwnText(messages("ated.property-details-period.taxAvoidancePromoterReference.error.empty")).hasText mustBe true
       errorDoc.getElementsMatchingOwnText(messages("ated.property-details-period-error.general.taxAvoidancePromoterReference")).hasText mustBe true
     }
+
+    "check error message for empty form" in {
+      doc.getElementsContainingOwnText(messages("ated.property-details-period-error.general.isFullPeriod")).hasText mustBe true
+      doc.getElementsContainingOwnText(messages("ated.property-details-period.isTaxAvoidance.error-field-name")).hasText mustBe true
```

Comment:
can you please use haveErrorSummary and haveErrorNotification from JsoupMatchers. It checks for both error Text and position of errors

Reply:
Fixed it